docs/docusaurus/versioned_docs/version-1.6.0/orc8r/deploy_intro.md

```diff
@@ -74,7 +74,7 @@ Verified with Terraform version `0.15.0`.
 - `v1.6` [patch branch](https://github.com/magma/magma/tree/v1.6)
 - `github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-aws?ref=v1.6`
 Terraform module source
-- `1.5.23` Helm chart version
+- `1.5.24` Helm chart version
```

Comment:
Do we have a strong reason to backport this chart change? If not, let's leave the backport out as it makes things messier

Reply:
Oh! Versioned docs. I wasn't looking carefully at the path. I'll fix this